plpgsql/pgunit/run_tests.sql

```diff
@@ -0,0 +1,97 @@
+CREATE OR REPLACE FUNCTION test_case_update_quality() RETURNS void AS $$
+DECLARE
+  sell_in_result item.sell_in%TYPE;
+  quality_result item.quality%TYPE;
+BEGIN
+  TRUNCATE TABLE item;
+  CALL new_item('Aged Brie', 4, 6);

```

Comment:
This example test is a little more extensive than the one I usually include in a starting position. I normally have an item called "foo" and an assertion that it is called "fixme" after update quality is called. I prefer this starting test since it fail, and gives the person doing the exercise a big clue as to what to do to make it pass. Then the person doing the exercise has to work out what a better test to write would be. Can you explain why you prefer to start with this test instead?